static/css/cf-theme-overrides.less

```diff
@@ -121,6 +121,9 @@
 // .error
 @input-error:                   @redorange;
 
+// .warning
+@input-warning:                 @gold;
```

Comment:
There are already success and error states for inputs. The code in `cf-enhancements` extends that to include warnings. That color should become a non-bureau default when ported back to CF and this one will over-ride that. Rather than have that happen and forget, I opted to just over-ride it with the same value for the time being.